TripHouse/index.html

```diff
@@ -189,15 +187,83 @@ <h2 class="homes-guests-text">Homes guests loves</h2>
             <p class="cities-text">Berlin, Germany</p>
           </li>
         </ul>
-        <svg class="button-homes" width="48" height="48">
+        <a class="button-homes-link" href="">
+          <svg class="button-homes" width="48" height="48">
+            <use href="../images/svg-sprite.svg#Button-homes"></use>
+          </svg>
+        </a>
+      </div>
+    </section>
+    <section class="best-dest-cont">
+      <h2 class="best-dest-heading">The best destinations</h2>
+      <div class="best-dest-but-cont">
+        <button class="best-dest-but" type="submit">Regions</button>
+        <button class="best-dest-but" type="submit">Cities</button>
+        <button class="best-dest-but" type="submit">Places of interest</button>
+        <button class="best-dest-but" type="submit">Places</button>
+      </div>
+      <ul class="best-dest-list">
+        <li class="best-dest-item Maldives">
+          <div class="maldives-wrapper">
+            <img class="pic1" src="../images/maldives-pic.svg" alt="Maldives" />
+            <button class="book-now" type="submit">Book now</button>
+          </div>
+          <p class="best-dest-text">Maldives</p>
+        </li>
+        <li class="best-dest-item Spain">
+          <img src="../images/spain-pic.svg" alt="Spain" />
+          <p class="best-dest-text">Spain</p>
```

Comment:
You should add the button **book now** for **every** item, and you should show this button on **hover**